resources/web/wwi/nodes/WbDevice.js

```diff
@@ -57,7 +57,9 @@ export default class WbDevice extends WbSolid {
     }
 
     currentNode.postFinalize();
-    const radiusScale = currentNode.boundingSphere().radius;
+    let boundingSphere = currentNode.boundingSphere();
```

Comment:
`boundingSphere` is modified by a method called at line 61.
So it seems wrong to set it as `const`.

Reply:
I think it is ok, because in JS a `const` is :
The value of a constant can't be changed through reassignment (i.e. by using the [assignment operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment)), and it can't be redeclared (i.e. through a [variable declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations)). However, if a constant is an [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object) or [array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) its properties or items can be updated or removed. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const)